spack/packages/dla-future/package.py

```diff
@@ -93,15 +94,18 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
             depends_on(
                 "rocblas amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val)
             )
+            depends_on("whip amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val))
             depends_on(
-                "rocprim amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val)
+                "umpire amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val)
             )
+
+    with when("@:0.3 +rocm"):
```

Comment:
In case, it applies here too.

```suggestion
    with when("@:0.3.1 +rocm"):
```